modules/mobile/spec/request/veterens_affairs_eligibility_request_spec.rb

```diff
@@ -0,0 +1,275 @@
+             { 'name' => 'audiology',
+               'requestEligibleFacilities' => ['489'],
+               'directEligibleFacilities' => ['489'] },
+             { 'name' => 'covid',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'optometry',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'outpatientMentalHealth',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'moveProgram',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'foodAndNutrition',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'clinicalPharmacyPrimaryCare',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => '411',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'primaryCare',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'homeSleepTesting',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'socialWork',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] }]
+          )
+        end
+      end
+
+      context 'multiple facilities' do
+        let(:params) { { facilityIds: %w[489 984] } }
+
+        before do
+          VCR.use_cassette('va_eligibility/get_scheduling_configurations_200', match_requests_on: %i[method uri]) do
+            get '/mobile/v0/appointments/va/eligibility', params: params, headers: iam_headers
+          end
+        end
+
+        it 'returns successful response' do
+          expect(response).to have_http_status(:success)
+        end
+
+        it 'matches schema' do
+          expect(response.body).to match_json_schema('service_eligibility')
+        end
+
+        it 'response properly assigns facilities to services' do
+          services = response.parsed_body.dig('data', 'attributes', 'services')
+
+          expect(services).to eq(
+            [{ 'name' => 'amputation',
+               'requestEligibleFacilities' => ['489'],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'audiology',
+               'requestEligibleFacilities' => ['489'],
+               'directEligibleFacilities' => %w[489 984] },
+             { 'name' => 'covid',
+               'requestEligibleFacilities' => [],
+               'directEligibleFacilities' => [] },
+             { 'name' => 'optometry',
+               'requestEligibleFacilities' => ['489'],
```

Comment:
Why do the results for 489 not match the `singe facility` context above? I would expect those to be the same.

Reply:
The original VCR I copied only had one or two eligible services which I didn't feel like put the code through its paces well enough so I enabled some more but didn't think to keep it consistent between the single and multiple. fixed now.

Reply:
Sounds like I must've misread something. I thought both contexts were using the same VCR.